CHANGELOG.md

```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## Windows/Linux/macOS 5.2.0 - 2017-02-14
+Updated PECL release packages. Here is the list of updates:
+
+### Added
+- Added support for Always Encrypted with basic CRUD functionalities (see [here](https://github.com/Microsoft/msphpsql/wiki/Features#aebindparam))
+  - Support for Windows Certificate Store (use connection keyword ColumnEncryption)
+  - Support for inserting into and modifying an encrypted column
+  - Support for fetching from an encrypted column
+- Added support for PHP 7.2
+- Added support for MSODBC 17
+- Added support for Ubuntu 17 (requires MSODBC 17)
+- Added support for Debian 9 (requires MSODBC 17)
+- Added support for SUSE 12
+- Added Driver option to set the ODBC driver, Added "Driver" option, valid values are ODBC Driver 17 for SQL Server,ODBC Driver 13 for SQL Server, and ODBC Driver 11 for SQL Server
+  - The default driver is ODBC Driver 17 for SQL Server
+
+### Changed
+- Implementation of PDO::lastInsertId($name) to return the last inserted sequence number if the sequence name is supplied to the function ([lastInsertId](https://github.com/Microsoft/msphpsql/wiki/Features#lastinsertid))
+
+### Fixed
+- Issue [#555](https://github.com/Microsoft/msphpsql/issues/555) - Hebrew strings truncation (requires MSODBC 17)
+- Adjusted precisions for numeric/decimal inputs with Always Encrypted
+- Support for non-UTF8 locales in Linux and macOS
+- Fixed crash caused by executing an invalid query in a transaction (Issue [#434](https://github.com/Microsoft/msphpsql/issues/434))
+- Added error handling for using PDO::SQLSRV_ATTR_DIRECT_QUERY or PDO::ATTR_EMULATE_PREPARES in a Column Encryption enabled connection
+- Added error handling for binding TEXT, NTEXT or IMAGE as output parameter (Issue [#231](https://github.com/Microsoft/msphpsql/issues/231))
+- PDO::quote with string containing ASCII NUL character (Issue [#538]( https://github.com/Microsoft/msphpsql/issues/538))
+- Decimal types with no decimals are correctly handled when AE is enabled (PR [#544](https://github.com/Microsoft/msphpsql/pull/544))
+- BIGINT as an output param no longer results in value out of range exception when the returned value is larger than a maximum integer ([PR #567](https://github.com/Microsoft/msphpsql/pull/567))
+
+### Removed
+- No longer support Ubuntu 15
+- Supplying tablename into PDO::lastInsertId($name) no longer return the last inserted row ([lastInsertId](https://github.com/Microsoft/msphpsql/wiki/Features#lastinsertid))
+
+### Limitations
+- Always Encrypted is not support in Linux and macOS
+- In Linux and macOS, setlocale() only takes effect if it is invoked before the first connection. The subsequent locale setting will not work
```

Comment:
'The subsequent locale setting will not work' - this is awkward. Maybe 'Attempting to set the locale after connection will not work'?